CMakeLists.txt

```diff
@@ -294,8 +353,8 @@ else()
 endif()
 
 if(DEFINED REF_BLAS_ROOT)
-  find_file(REF_BLAS_LIBNAME NAMES blas.dll libblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
-  find_file(REF_CBLAS_LIBNAME NAMES cblas.dll libcblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
+  find_file(ONEAPI_ONEMKL_REF_BLAS_LIBNAME NAMES blas.dll libblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
+  find_file(ONEAPI_ONEMKL_REF_CBLAS_LIBNAME NAMES cblas.dll libcblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
```

Comment:
We don't need to rename these occurrences in CMake files. There are a few other occurrences.

Reply:
Following applies also to the other [comment](https://github.com/oneapi-src/oneMKL/pull/571#discussion_r1763051305)

I think that changes the name here is better. `find_file()` function creates the variable we pass as first argument, so I think it is better to define only one variable and then use it, rather than define one variable here and then specify what we really want in another place.
A possibility is to move those line inside `src/CMakeLists.txt` with the other variables defined for the header generation, since these two variables have the same usage.